.teamcity/Gradle_Util_Performance/buildTypes/AdHocPerformanceScenario.kt

```diff
@@ -52,6 +52,7 @@ abstract class AdHocPerformanceScenario(os: Os) : BuildType({
         }
         gradleWrapper {
             name = "GRADLE_RUNNER"
+            workingDir = os.perfTestWorkingDir
```

Comment:
We need to run `subst` here as well before we use the drive as a working dir.

Reply:
Done.